pxtextension.ts

```diff
@@ -35,7 +31,7 @@ namespace mlrunner {
     }
     const actionLabels = actions.map((action, i) => ({
       name: action,
-      value: i + 1,
+      value: i + 2,
```

Comment:
Because "None" needs to have a non-zero value, the user labels start at value "2", so I assumed this needs to be increased, but I'm not able to test it.

Alternatively we could check if the events can take negative values and make None = -1, but this is easier.